src/rp2_common/hardware_pio/pio.c

```diff
@@ -232,8 +241,17 @@ void pio_sm_set_pins(PIO pio, uint sm, uint32_t pins) {
 }
 
 void pio_sm_set_pins_with_mask(PIO pio, uint sm, uint32_t pinvals, uint32_t pin_mask) {
+    pio_sm_set_pins_with_mask64(pio, sm, pinvals, pin_mask);
+}
+
+void pio_sm_set_pins_with_mask64(PIO pio, uint sm, uint64_t pinvals, uint64_t pin_mask) {
     check_pio_param(pio);
     check_sm_param(sm);
+    check_pio_pin_mask64(pio, pin_mask);
```

Comment:
Should it also do `check_pio_pin_mask64(pio, pinvals);` ?

Reply:
no